internal/ethapi/api.go

```diff
@@ -138,18 +138,9 @@ func (s *EthereumAPI) Syncing() (interface{}, error) {
 	return false, nil
 }
 
-type GetChainConfigResponse struct {
-	*params.ChainConfig
-	params.UpgradeConfig `json:"upgrades"`
-}
-
-func (s *BlockChainAPI) GetChainConfig(ctx context.Context) GetChainConfigResponse {
-	config := s.b.ChainConfig()
-	resp := GetChainConfigResponse{
-		ChainConfig:   config,
-		UpgradeConfig: config.UpgradeConfig,
-	}
-	return resp
+// GetChainConfig returns the chain config.
+func (s *EthereumAPI) GetChainConfig(ctx context.Context) *params.ChainConfig {
```

Comment:
duplicated API?

Reply:
I am not sure how this is supposed to work? should this be defined on EthereumAPI or BlockchainAPI?

Reply:
ChainID is under the BlockchainAPI so I thought it makes sense to put this under BlockchainAPI.

Reply:
I think this makes more sense under `BlockchainAPI` and we should make sure that it still includes the `UpgradeConfig` change so that the upgrade bytes are included in the JSON response correctly

Reply:
Should we make the same change in coreth?

Reply:
Ah I see, yes we should, we actually documented it this way as well: https://docs.avax.network/apis/avalanchego/apis/subnet-evm#eth_getchainconfig